### DISCUSSION TITLE
Ability To Browse By Static Text In Web Content

### DISCUSSION THREAD

**Discussion by ghost** [nvaccess/nvda#15062] [Ideas]
### Is your feature request related to a problem? Please describe.
No. This request is for improving navigability of web content.

### Describe the solution you'd like
As an NVDA user accessing web content frequently, I'd like a browse mode shortcut to move between static text elements, so that I can more easily locate where the content of an article, post or other document begins.
Often times, web pages contain various controls, links and advertisements such that it is hard to reliably bypass them and get to the next instance of regular static text. This enhancement would alleviate that challenge. 

### Describe alternatives you've considered
Currently, the N browse mode shortcut skips forward past a block of links. This behavior is seldom useful on the modern web, as plain text can follow any number of element types, not just links. Instead of navigating based on the presence of links, it should simply navigate to the next block of static text. For an example of this in action, Apple's VoiceOver screen reader for iOS and MacOS has a rotor option called "Static Text" which does exactly this, demonstrating that the functionality is achievable.

### Additional context
A good example of the need for this feature is when you're reading an article online. Nowadays there are usually several elements - headings, images and social links, for example, preceding the actual text of that article. Furthermore, articles are interrupted by an increasing number of advertisements. These ads are often in their own content containers and make navigation to the next part of the article hard or impossible. If we could simply move between plain text elements, this would greatly ease navigation of this type of content.

**Comment by seanbudd** [comment]
We'd like more feedback on this suggestion from the community
